grazer/src/main/java/edu/asu/diging/grazer/core/graphs/impl/GraphManager.java

```diff
@@ -65,17 +69,37 @@ public void init() {
         transformationNames.add("someone_has_sth_end_date");
         transformationNames.add("sth_has_so_end_date");
         transformationNames.add("someone_has_sth_occur_date");
-        transformationNames.add("sth_has_so_occur_date");
+        transformationNames.add("sth_has_so_occur_date");*/
+        
+        
+       if(folder.exists()) {
+	        	if(folder.isDirectory()) {
+	        		fileNames = folder.listFiles();
+	        		if(fileNames != null && fileNames.length > 0) {
+	        			//fileNames[0] contains a hidden file DS_Store, so starting the for loop with index 1
+	        			for(int i = 1; i < fileNames.length; i++) {
+	        				// Removing prefixes PAT_ and TRA_ and .graphml at the end 
```

Comment:
for each PAT there is a TRA file, so you don't need to loop over all files (that just doubles the loop).